src/yadism/esf/tmc.py

```diff
@@ -46,22 +49,33 @@ def g2_ker(z, _args):
     return 1 - z
 
 
+@nb.njit("f8(f8,f8[:])", cache=True)
+def k1_ker(_z, _args):
+    # TODO: Check that following makes sense
+    return 1
```

Comment:
@felixhekhorn This is what I was alluding to, although looking at the ration doesn't hint that something is wrong.

![pred_g1_total](https://user-images.githubusercontent.com/43742664/235639962-3398ce0c-ba9f-4580-856c-71cd9f5705ca.png)

Reply:
Actually, there seems to be a bug for F3 (cc @AleCandido ):
https://github.com/NNPDF/yadism/blob/d947168752d48636a1c0fa0cc6709c27150b81bd/src/yadism/esf/tmc.py#L526-L528
but we're not doing what we promise:
https://github.com/NNPDF/yadism/blob/d947168752d48636a1c0fa0cc6709c27150b81bd/src/yadism/esf/tmc.py#L44
instead it should be `h3_ker = k1_ker`

In any case `_convolute_FX` should do a convolution as I said ...

Reply:
@Radonirinaunimi can you double check this? and fix if you agree